### PR TITLE
edit profile interface and new avatars

### PR DESCRIPTION
**Pull request by quiz1991**
1. edit profile ui interface
2. imported new avatars and pick default one according to userId
3. update in db upon submit(only username now since we do not have firstname and lastname fields in db)

@anthonyseu @jacgrady1 

screen shot 2017-06-18 at 8 29 33 am

### CODE REVIEW

File: actions/ProfileModalActions.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export default {
+    closeAnswerModal(context, payload, done) {
+        context.dispatch('PROFILE_MODAL_CLOSE');
+        done();
+        return;
+    },
+    openModalWithPayload(context, payload, done) {
+        console.log('feqwopfjhpoqew jp9 iqjwepoif jpqoiwejfpo');
```
Comment (anthonyseu) [inline actions/ProfileModalActions.js]: nit: can you remove the log?

File: components/partials/ProfileModal.js [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+import { tftGreyDark } from '../../config/colorConstants';
+
+class ProfileModal extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {};
+        this.handleModalClose = this.handleModalClose.bind(this);
+        this.handleModalSubmit = this.handleModalSubmit.bind(this);
+    }
+    // componentWillReceiveProps(nextProps){
```
Comment (anthonyseu) [inline components/partials/ProfileModal.js]: nit: can you remove the comment

**Comment by jacgrady1** [comment]
screenshot?

File: actions/ProfileModalActions.js (hunk 1 above)
Comment (quiz1991) [inline actions/ProfileModalActions.js]: sure :P

File: components/partials/ProfileModal.js (hunk 2 above)
Comment (quiz1991) [inline components/partials/ProfileModal.js]: sure :P

**Comment by quiz1991** [comment]
@jacgrady1 updated with ss

**Comment by quiz1991** [comment]
@jacgrady1 @anthonyseu 

added updating db upon submit(only taking username now since we do not have firstname and lastname fields in db)

**Comment by jacgrady1** [comment]
cool.

**Comment by anthonyseu** [comment]
I think you can create a db migration and add first/last name field

**Comment by anthonyseu** [comment]
> I think you can create a db migration and add first/last name field

No worries, let's do it later.

**Pull request merged**